### PR TITLE
2.7.0 Blog Post

### PR DESCRIPTION
**Pull request by droidmonkey**
For review prior to posting and sharing

### CODE REVIEW

File: blog/_posts/2022-03-21-2.7.0-released.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+category: "Releases"
+permalink: /:path/
+author: Jonathan White
+teaser_image: /images/keepassxc-logo.svg
+teaser_image_alt: "KeePassXC logo"
+excerpt_separator: <!--more-->
+---
+
+Today we are releasing **KeePassXC 2.7.0**. Closing 118 issues and merging in 149 pull requests, this 
+release is one of our biggest and most important yet! We have incorporate several significant improvements
```
Comment (louib) [inline blog/_posts/2022-03-21-2.7.0-released.md:14]: @droidmonkey `incorporated`

File: blog/_posts/2022-03-21-2.7.0-released.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+including:
+
+* Entry tagging
+* Quick unlock using Windows Hello and macOS Touch ID
+* Massive overhaul of Auto-Type
+* Improvements to attachment handling
+* Improvements to Secret Service (Linux)
```
Comment (louib) [inline blog/_posts/2022-03-21-2.7.0-released.md:21]: @droidmonkey Secret Service integration?

File: blog/_posts/2022-03-21-2.7.0-released.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* Convert crypto backend to [Botan](https://github.com/randombit/botan)
+* Much much more...
+
+<!--more-->
+
+Pictures are worth a thousand words, so here are some shots of the new features within 2.7.0. The first
+image shows the new Auto-Type selection dialog brimming with options. You can now easily search all open
+databases for an entry to type using the same powerful search syntax as the main application. You can also
+type out TOTP codes and copy data to the clipboard from this dialog. We feel like this is a game changer
+to interacting with your database.
```
Comment (louib) [inline blog/_posts/2022-03-21-2.7.0-released.md:31]: @droidmonkey game changer `when` interacting with your db?

Comment (phoerious) [reply blog/_posts/2022-03-21-2.7.0-released.md:31]: Game changer to + gerund sounds fine to me.

**Pull request merged**